system-parachains/coretime/coretime-kusama/src/coretime.rs

```diff
@@ -183,15 +255,27 @@ impl CoretimeInterface for CoretimeAllocator {
 		}
 	}
 
-	fn check_notify_revenue_info() -> Option<(RCBlockNumberOf<Self>, Self::Balance)> {
-		let revenue = CoretimeRevenue::get();
-		CoretimeRevenue::set(&None);
-		revenue
-	}
+	fn on_new_timeslice(t: pallet_broker::Timeslice) {
+		// Burn roughly once per day. Unchecked math: RHS hardcoded as non-zero.
+		if t % 180 != 0 {
```

Comment:
180 probably should base on timeslice value, otherwise just a magic value in this context

Reply:
Yea i also wondered about this. @seadanda why is it hard-coded?

Reply:
It was just a last minute addition in @s0me0ne-unkn0wn PR to this one. It can be generalised for sure.

Reply:
Resolved in https://github.com/polkadot-fellows/runtimes/pull/410/commits/09a73f451c7b93dc052dea43a706236677c15fc9